src/clr-angular/data/datagrid/datagrid-column-separator.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component, Renderer2 } from '@angular/core';
+
+import { ColumnResizerService } from './providers/column-resizer.service';
+import { TableSizeService } from './providers/table-size.service';
+
+let nbCount: number = 0;
+
+@Component({
+  selector: 'clr-dg-column-separator',
+  template: `
+    <button class="datagrid-column-handle" tabindex="-1" type="button"
+      clrDraggable 
+      [clrGroup]="columnSeparatorId" 
+      (clrDragStart)="columnResizerService.startResize(); showTracker(resizeTrackerEl)" 
+      (clrDragMove)="columnResizerService.calculateResize($event); moveTracker(resizeTrackerEl)" 
+      (clrDragEnd)="columnResizerService.endResize(); hideTracker(resizeTrackerEl)"></button>
+    <div class="datagrid-column-resize-tracker" #resizeTrackerEl></div>
+    `,
+  host: {
+    '[class.datagrid-column-separator]': 'true',
+  },
+})
+export class ClrDatagridColumnSeparator {
+  // Every column draggable separator should have its own unique ID
+  // in order to not conflict with other draggables/droppables.
+  public columnSeparatorId: string;
+
+  constructor(
+    public columnResizerService: ColumnResizerService,
```

Comment:
Make this private and call the resizer service inside of your show/move/hide tracker methods instead. Don't want to expose this internal service if we don't need to.